src/sections/perps/perpetuals.tsx

```diff
@@ -0,0 +1,190 @@
+import { useState } from 'react';
+import { Flex, FlexProps, Text, Button, Link } from '@chakra-ui/react';
+import { PerpetualSynth } from 'pages/perps';
+import FutureSynthCard from 'src/components/FutureSynthCard';
+
+interface PerpetualsProps extends FlexProps {
+	synths: PerpetualSynth[];
+}
+
+const perpV2Synths = ['sETH'];
+
+export default function Perpetuals({ synths, ...props }: PerpetualsProps) {
+	const [activeTab, setActiveTab] = useState('all');
+	const [activeVersion, setActiveVersion] = useState<'V1' | 'V2'>('V1');
+
+	let tabs: string[] = ['all'];
+
+	const synthsDictionaryV1: Record<string, JSX.Element[]> = {
+		all: [],
+	};
+
+	const synthsV2: PerpetualSynth[] = [];
```

Comment:
Feels a little wrong to create the dictionaries in the render fn, I would extract to a function. But I guess you just followed the already existing patterns

Reply:
The synths are being passed as a prop - I think once we have a list for both we can update this pattern.